ios/brave-ios/Sources/BraveStrings/BraveStrings.swift

```diff
@@ -9183,6 +9183,16 @@ extension Strings {
       value: "Brave Rewards",
       comment: ""
     )
+
+    public static let braveAdsDataToggleOption =
+      NSLocalizedString(
+        "ads.braveAdsDataToggleOption",
+        tableName: "BraveShared",
+        bundle: .module,
+        value: "Brave Ads Data",
+        comment:
+          "Text for Brave Ads Data Toggle while clearing the data in settings"
```

Comment:
Is "Ads" here supposed to be translated? Can you be specific in the comment for the translator to ensure that at least "Brave" or "Brave Ads" is the product name and shouldn't be translated and that data refers to information stored on device

Reply:
`Brave Ads` shouldn't be translated. I've changed comment to `Label for the toggle switch to clear 'Brave Ads' data in settings`